packages/bootstrap/scss/pager/_theme.scss

```diff
@@ -1,36 +1,6 @@
 @import "~@progress/kendo-theme-default/scss/pager/_theme.scss";
 
 @include exports( "pager/theme/bootstrap" ) {
```

Comment:
here we can remove the whole mixin since we're not adding any bootstrap specific styles